Networking/NetworkingTests/Responses/me-settings.json

```diff
@@ -0,0 +1,61 @@
+{
+	"enable_translator": false,
+	"surprise_me": false,
+	"post_post_flag": false,
+	"holidaysnow": false,
+	"user_login": "test123123",
+	"password": "",
+	"display_name": "test123123",
+	"first_name": "Dem 123",
+	"last_name": "Nines",
+	"description": "🙃\n\nThis is a bit more about me.\n\nAnd multi line text.",
+	"user_email": "moocow@example.com",
+	"user_email_change_pending": false,
+	"new_user_email": "",
+	"user_URL": "http://www.xcode.wtf",
```

Comment:
🤨 how about `.mmm` instead of `.wtf` ?

Reply:
LOL I had forgotten I had that domain associated with my test account. To be fair it is a real domain and I own it. 😂🤣 I'll change it to something less offensive for sure!!